### PR TITLE
story/SFO-43_-_Buil-Hero_image

### PR DESCRIPTION
**Pull request by JorgeRojano**
Hero image built

### CODE REVIEW

File: Jorge/css/hero-image.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  font-size: 64px;
+  margin: 0;
+}
+.hero-text span {
+  color: blue;
+  font-size: 32px;
+  background-color: wheat;
+  padding: 8px;
+}
+.hero-text button {
```
Comment (frank-martinez-iest) [inline Jorge/css/hero-image.css]: Using BEM correctly would remove the need to create `classname element` style blocks, instead you should set a classname on each element such as `hero-text__title`, `hero-text__title-highlight`, `hero-text__button`, and so on

File: Jorge/index.html [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        <div class="top-nav__phone">
+          <i class="fa fa-phone" aria-hidden="true"></i>
+          (123) 456-7890
+        </div>
+        <div class="top-nav__email">
+          <i class="fa fa-envelope-o" aria-hidden="true"></i>
+          contacts@mail.com
+        </div>
+      </div>
+      <div class="top-nav__social-icons">
```
Comment (frank-martinez-iest) [inline Jorge/index.html]: This should be a UL

File: Jorge/index.html [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+        <i class="fa fa-twitter" aria-hidden="true"></i>
```
Comment (frank-martinez-iest) [inline Jorge/index.html]: Each of these should be wrapped in an LI and have an anchor so when the user clicks on each they are taken to your social networks, right?

**Review by frank-martinez-iest — CHANGES_REQUESTED** [review]
Great job! Left few comments

File: Jorge/index.html (hunk 2 above)
Comment (JorgeRojano) [reply Jorge/index.html]: Solved

File: Jorge/index.html (hunk 3 above)
Comment (JorgeRojano) [reply Jorge/index.html]: Done

**Pull request merged**